demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -75,30 +77,32 @@ def _run_query(self, playground_id: str):
             'investigationId': playground_id,
             'data': self.query
         }
-        ans = self.client.investigation_add_entries_sync(update_entry=update_entry)
+
+        answer = self.client.investigation_add_entries_sync(update_entry=update_entry)
+        if not answer:
+            raise RuntimeError
```

Comment:
don't raise a class, raise an object (with a proper descriptive message)

Reply:
correct.
fixed it, error looks ok.
```
(py37) ➜  content git:(securonix1_assign) ✗ demisto-sdk run -q 'securonix-list-incidentsss'
You are using demisto-sdk 0.4.9.dev14, however version 0.4.8 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
Command did not run, make sure it was written correctly.
```